db/migrate/20200526200555_add_gear_to_playbook.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
Can you remove this migration from the commit history?

Reply:
Removed in commit 6ef0a4a